content/enterprise_influxdb/v1.9/install-and-deploy/production_installation/meta_node_installation.md

```diff
@@ -10,8 +10,8 @@ menu:
 ---
 
 InfluxDB Enterprise offers highly scalable clusters on your infrastructure
-and a management UI ([via Chronograf](/{{< latest "chronograf" >}}) for working with clusters.
-The Production Installation process is designed for users looking to
+and a management UI ([via Chronograf](/{{< latest "chronograf" >}})) for working with clusters.
+The production installation process is designed for users looking to
```

Comment:
We can drop the "production installation" reference here (used to have something called the "quickstart installation" but dropped that so it's all now "installation." 

Perhaps we replace lines 14-17 w something like:

To install InfluxDB Enterprise meta nodes, do the following:
1. Review meta node requirements
2. Set up meta nodes:
     a. Add DNS entries...
     b. Set up, configure, and start the meta services
     c. Join meta nodes to the cluster

Reply:
Also looks like we have a mix of roman numerals, numbers, and nested sequential lists without step "a", so we'll want to clean up the hierarchy across the board.